reframe/frontend/cli.py

```diff
@@ -1394,7 +1396,6 @@ def module_unuse(*paths):
             log_files = logging.log_files()
             if site_config.get('general/0/save_log_files'):
                 log_files = logging.save_log_files(rt.output_prefix)
-
```

Comment:
I think we still want this empty line, right?

Reply:
No, because in the line afterwards, we continue one level up.